### ISSUE TITLE
How to use similarity(contrastive) loss and train model with binary-cross-entropy as in many siamese(two branch CNN) based models .. needs to add a new loss function and prepare labelled pairs ? 

### ISSUE THREAD

**Issue by bmiftah** [KaiyangZhou/deep-person-reid#239]

**Comment by KaiyangZhou** [comment]
for siamese training, you can mimic the triplet code

i am closing this issue for now

**Comment by bmiftah** [comment]
Thank you and I will check out but why close the issue so fast ?

**Comment by KaiyangZhou** [comment]
I am quite busy so I am unable to answer too many technical questions. I usually hope sb can help with this. As this thread is inactive for some time so I close it

I am happy to reopen it if you want :)
